### PR TITLE
Add Discourse::AddUsersToGroupService

### PR DESCRIPTION
**Pull request by piotrekpasciak**
Monday:
https://aicrowd.monday.com/boards/363093650/pulses/523058269

### CODE REVIEW

File: app/services/discourse/add_users_to_group_service.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+module Discourse
+  class AddUsersToGroupService < ::Discourse::BaseService
```
Comment (skbly7) [inline app/services/discourse/add_users_to_group_service.rb:2]: Where is this service being called from?

File: app/services/discourse/create_group_service.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+module Discourse
+  class CreateGroupService < ::Discourse::BaseService
```
Comment (skbly7) [inline app/services/discourse/create_group_service.rb:2]: This seems to be already going as part of #1348 PR?

File: app/services/discourse/add_users_to_group_service.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    def add_users_to_group_request
+      client.put(
+        "/groups/#{challenge.discourse_group_id}/members.json", {
+          usernames: participants_usernames
+        }
+      )
+    end
+
+    def participants_usernames
+      participants.pluck(:name).join(',')
```
Comment (skbly7) [inline app/services/discourse/add_users_to_group_service.rb:38]: `@participants` or `participants` ?

File: app/services/discourse/add_users_to_group_service.rb (hunk 1 above)
Comment (piotrekpasciak) [reply app/services/discourse/add_users_to_group_service.rb:2]: I don't call it yet. I'll implement business logic in later PR's.

File: app/services/discourse/create_group_service.rb (hunk 2 above)
Comment (piotrekpasciak) [reply app/services/discourse/create_group_service.rb]: I've rebased with master and resolved the duplication

File: app/services/discourse/add_users_to_group_service.rb (hunk 3 above)
Comment (piotrekpasciak) [reply app/services/discourse/add_users_to_group_service.rb:38]: `participants` - I have attr_reader defined above.

**Pull request merged**